File: VB/WindowsApplication1/MyLayoutControl/MyPageMinSizeHelper.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraPrinting.Export
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.LookAndFeel

Namespace WindowsApplication1
	<System.ComponentModel.DesignerCategory("")> _
	Public Class MyPageMinSizeHelper
		Inherits Component

		Public Shared PageMinimumSize As New Size(200, 50)

		Public Property PageMinSize() As Size
			Get
				Return PageMinimumSize
			End Get
			Set(ByVal value As Size)
				If PageMinimumSize <> value Then
					PageMinimumSize = value
				End If
				LookAndFeelHelper.ForceDefaultLookAndFeelChanged()
			End Set
		End Property

		Public Sub New()
			DevExpress.XtraTab.Registrator.PaintStyleCollection.DefaultPaintStyles.Remove("Skin")
			DevExpress.XtraTab.Registrator.PaintStyleCollection.DefaultPaintStyles.Add(New MySkinViewInfoRegistrator())
			LookAndFeelHelper.ForceDefaultLookAndFeelChanged()
		End Sub
	End Class
End Namespace

File: CS/Demo/WindowsApplication1/MyLayoutControl/MyPageMinSizeHelper.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraPrinting.Export;
using DevExpress.XtraTab.Registrator;
using DevExpress.LookAndFeel;

namespace WindowsApplication1
{
    [System.ComponentModel.DesignerCategory("")]
    public class MyPageMinSizeHelper : Component
    {

        public static Size PageMinimumSize = new Size(200, 50);

        public Size PageMinSize
        {
            get { return PageMinimumSize; }
            set
            {
                if (PageMinimumSize != value)
                    PageMinimumSize = value;
                LookAndFeelHelper.ForceDefaultLookAndFeelChanged();
            }
        }

        public MyPageMinSizeHelper()
        {
            DevExpress.XtraTab.Registrator.PaintStyleCollection.DefaultPaintStyles.Remove("Skin");
            DevExpress.XtraTab.Registrator.PaintStyleCollection.DefaultPaintStyles.Add(new MySkinViewInfoRegistrator());
            LookAndFeelHelper.ForceDefaultLookAndFeelChanged();
        }
    }
}
